components/openzwave-adapter/src/lib.rs

```diff
@@ -352,6 +385,12 @@ impl OpenzwaveAdapter {
                         if kind.is_none() { continue }
                         let kind = kind.unwrap();
 
+                        let mut tags = HashSet::new();
+                        let label = vid.get_label();
+                        if !label.is_empty() {
+                            tags.insert(TaxoId::<TagId>::new(&vid.get_label()));
```

Comment:
Could you please clarify what this label is about exactly?

I'm currently working on the UI where user can add/remove tags (for the services for now), is this `label` the thing that can be safely removed by the user without losing something very important?